### ISSUE TITLE
Groups Toolbar Not Frozen Although User Has Partial Permission

### ISSUE THREAD

**Issue by LeoNerdoG** [eclipse/kapua#1027]
Problem emerges when creating a user (e.g. kapua-user1) and granting Group permissions. Although we grant partial permissions (only read/write/delete or any combination of read and others), group toolbar of kapua-user1 is not frozen (in every case **add**, **edit** and **delete** options are clickable).
If user tries to perform an action for which he does not have permissions Kapua returns an error (which is ok).

Testflow:
1. Login as kapua-sys
2. Go to Users, create a user (e.g. kapua-user1)
3. Select kapua-user1, add read or read and write/delete/ Groups permissions
4. Login as kapua-user1 and try to read/edit/delete groups.
**Basically permissions work, the only problem is that toolbar is not frozen as it should be.**

![screen shot 2017-11-22 at 10 22 38](https://user-images.githubusercontent.com/15121999/33120584-cbef31b0-cf72-11e7-9a3f-713e2f23e007.png)

Browsers used:
- Google Chrome (Version 62.0.3202.94 (Official Build) (64-bit))
- Safari (Version 11.0.1 (13604.3.5))
- Mozilla Firefox (57.0 (64-bit))
- Opera (Version 49.0.2725.39)

**Comment by lorthirk** [comment]
#1432 covers the generic case for the Add button

**Comment by LeoNerdoG** [comment]
I have verified that this issue has been resolved with PR #1525.
